server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -243,9 +243,7 @@ public void testNestedLoop_TablesAreSwitchedIfBlockJoinAndRightIsSmallerThanLeft
 
     @Test
     public void testNestedLoop_TablesAreNotSwitchedIfLeftHasAPushedDownOrderBy() {
-        // we use a subselect to simulate the pushed-down order by
-        QueriedSelectRelation mss = e.analyze("select users.id from (select id from users order by id) users, " +
-                                              "locations where users.id = locations.id");
+        QueriedSelectRelation mss = e.analyze("select users.id from users, locations where users.id = locations.id order by users.id");
```

Comment:
The pushed-down order by is now determed by the property `nestedLoop.orderByWasPushedDown()` and it therefore needed to be pushed down before to make that test work.